core/arch/arm/plat-imx/config/config_imx6sx.h

```diff
@@ -21,7 +23,7 @@
 #define DRAM0_SIZE		CFG_DDR_SIZE
 
 /* Location of trusted dram */
-#define TZDRAM_BASE		(DRAM0_BASE - 32 * 1024 * 1024 + CFG_DDR_SIZE)
+#define TZDRAM_BASE		(CFG_DDR_START - 32 * 1024 * 1024 + CFG_DDR_SIZE)
```

Comment:
Previous implementation relied on hardcoded `DRAM0_START`. Restore it and don't introduce a new `CFG_DDR_START` directive.